scopus-import/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -116,7 +116,7 @@ public ImportCandidate generateImportCandidate() {
                    .withResourceOwner(constructResourceOwner())
                    .withAdditionalIdentifiers(generateAdditionalIdentifiers())
                    .withEntityDescription(generateEntityDescription())
-                   .withModifiedDate(Instant.now())
+                   .withCreatedDate(Instant.now())
```

Comment:
Generally, the modified date is set to the created date when initialised, but this maybe isn't relevant